src/files-card-body.tsx

```diff
@@ -127,6 +132,9 @@ export const FilesCardBody = ({
                 .filter(file => file.name.toLowerCase().includes(currentFilter.toLowerCase()))
                 .sort(compare(sortBy));
     }, [files, showHidden, currentFilter, sortBy]);
+    const hiddenFilesCount = useMemo(() => {
+        return files.filter(file => file.name.startsWith(".")).length;
+    }, [files]);
```

Comment:
Totally not a critique, just for my own education: this operation feels quite cheap to me, unless we are talking about millions of files. Is the overhead of a memo justified for that?